config/form-content/bestuursperiode/form.ttl

```diff
@@ -1,64 +0,0 @@
-@prefix form: <http://lblod.data.gift/vocabularies/forms/>.
```

Comment:
this entire form was just plain wrong (types, predicates). Luckily it was also unused.